### DISCUSSION TITLE
DateTime Quickfilter without range

### DISCUSSION THREAD

**Discussion by alexatcano** [serenity-is/Serenity#7286] [Help] [answered]
Hello,

I am using Serenity 5.2.3 (I know it is old, can't update , it's an "enterprise app" aka stuck in time)
and when using QuickFilter on a DateTime field it presents a range 
![image](https://github.com/user-attachments/assets/d2e3a2ec-084c-48c2-bb05-6b090a86f4b4)

How can i get rid of the range?

**Comment by VictorTomaili** [comment, marked as answer]
Check our sample about quick filter customization. https://demo.serenity.is/AdvancedSamples/QuickFilterCustomization

With similar approach (for old version you may need to checkout older commit to see code) you can add datetime editor for quick filter and modify the grid request criteria. 

`create a range editor for freight` sample demonstrate how to add new custom quick filter to your grid. Instead `DecimalEditor` you need to use DateEditor and in criteria you need to send criteria where is the date is bigger or less than the selected date. 
https://github.com/serenity-premium/StartSharp/blob/master/pro-features/src/Serenity.Demo.AdvancedSamples/Modules/Grids/QuickFilterCustomization/QuickFilterCustomizationPage.tsx

**Reply by VictorTomaili** [reply]
I just realized this is in serenity discussion section :) even you don't have startsharp, the idea is similar. Add your quick filter manually in your grid and update the criteria of the request.

**Reply by alexatcano** [reply]
Hi! actually we did buy a startsharp license.

**Reply by VictorTomaili** [reply]
So, next time you can ask the question in StartSharp repository 👍
